test/functional/api/autocmd_spec.lua

```diff
@@ -209,7 +209,7 @@ describe('autocmd api', function()
 
       local aus = meths.get_autocmds({ event = 'User', pattern = 'Test' })
       local first = aus[1]
-      eq(first.id, 1)
+      eq(true, first.id > 0)
```

Comment:
just for future reference: can use ok() to get a nicer failure message.
```suggestion
      ok(first.id > 0, 'first.id > 0', first.id)
```